src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
+const sumAgeDifference = (sum, x) => sum + (x.died - x.born);
+const calculateAverageAge
+= (peopleArray) => peopleArray.reduce(sumAgeDifference, 0) / peopleArray.length;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = people.filter((x) => (arguments.length === 2)
```

Comment:
Replace `arguments.length === 2` with `century !== undefined` or just `century` for better readability and to avoid using the `arguments` object.